### PR TITLE
fix: Add Split parameter migration

### PR DESCRIPTION
**Pull request by fhuaulme**
**Link to the JIRA issue**
(no jira)

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [ ] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [x] No, and no need to (backend changes only)

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/158/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/158/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/159/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/159/)

File: dataprep-upgrade-common/src/main/java/org/talend/dataprep/upgrade/common/ActionNewColumnToggleCommon.java [hunk 1]
```diff
@@ -32,26 +28,10 @@ private ActionNewColumnToggleCommon() {
...
-                        LOGGER.debug("Migration changed action id from '{}' to '{}', updating steps", beforeUpdateId,
-                                afterUpdateId);
-                        preparationRepository.list(PersistentStep.class, eq("contentId", beforeUpdateId)) //
-                                .filter(s -> !Step.ROOT_STEP.id().equals(s.id())) //
-                                .peek(s -> s.setContent(afterUpdateId)) //
-                                .forEach(preparationRepository::add);
-                    }
-                }) //
-                .forEach(preparationRepository::add); //
+        ParameterMigration.upgradeParameters(preparationRepository, //
```
Comment (aleboulanger) [inline dataprep-upgrade-common/src/main/java/org/talend/dataprep/upgrade/common/ActionNewColumnToggleCommon.java]: You can use `ParameterMigration.upgradeParameters(PreparationRepository repository, Consumer<Action> updater)`

Comment (fhuaulme) [reply dataprep-upgrade-common/src/main/java/org/talend/dataprep/upgrade/common/ActionNewColumnToggleCommon.java]: 👍

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/164/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/164/)

**Pull request merged**